clang/include/clang/Basic/AttrDocs.td

```diff
@@ -396,9 +396,12 @@ any option of a multiversioned function is undefined.
   }];
 }
 
-def SYCLKernelDocs : Documentation {
+def DeviceKernelDocs : Documentation {
   let Category = DocCatFunction;
+  let Heading = "device_kernel";
```

Comment:
We should probably list the other spellings of the attribute to the heading, that helps folks searching for those docs.

Reply:
done in latest commit, thanks!